src/components/BalanceModal/BalanceModal.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { Box, IconButton, Modal, Stack, Typography } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import { Form, Formik } from 'formik';
+import * as yup from 'yup';
+import { LoadingButton as Button } from '@mui/lab';
+import Input from '../Input';
+import Translate from '../Translate';
+
+const BalanceModal: FC<{
+  handleClose: () => void;
+  handleSubmit: (value: { amount: number }) => void;
+  modalOpened: boolean;
+  loading: boolean;
+}> = ({ handleClose, handleSubmit, modalOpened, loading }) => (
+  <Modal
```

Comment:
You also can use https://mui.com/material-ui/react-dialog/

Reply:
Thanks) I even had to refresh my understanding of the difference between modal and dialog.